core/db_models/EEM_Country.model.php

```diff
@@ -184,7 +184,7 @@ public function get_all_active_countries(): array
      * @return string
      * @throws EE_Error
      */
-    public function get_country_name_by_ISO(string $country_ISO): string
+    public function get_country_name_by_ISO(string $country_ISO)
```

Comment:
think you'll need to remove the `string` type hinting too, as I believe that was only added in PHP 7.0

```suggestion
    public function get_country_name_by_ISO($country_ISO)
```